haystack_experimental/components/retrievers/chat_message_retriever.py

```diff
@@ -39,14 +39,19 @@ class ChatMessageRetriever:
     ```
     """
 
-    def __init__(self, message_store: ChatMessageStore):
+    def __init__(self, message_store: ChatMessageStore, last_k: int = 10):
         """
         Create the ChatMessageRetriever component.
 
         :param message_store:
             An instance of a ChatMessageStore.
+        :param last_k:
+            The number of last messages to retrieve. Defaults to 10 messages if not specified.
         """
         self.message_store = message_store
+        if last_k <= 0:
+            raise ValueError(f"last_k must be greater than 0. Currently, the last_k is {last_k}")
+        self.last_k = last_k
```

Comment:
Can there be cases where the user might want to set `last_k=0` to not retrieve any chat messages in that particular run or thats completely invalid? Just a thought 🤔

Reply:
Hm, good point. Perhaps there could be cases like the one you describe. I'll adjust this check 🙏